.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-sudo: false
```

Comment:
@alcohol but it makes it harder to copy-paste the config elsewhere (composer has some repository where it is not the default)